pkgs/development/interpreters/emilua/default.nix

```diff
@@ -53,22 +53,21 @@ stdenv.mkDerivation rec {
     luajit_openresty
     re2c
     gperf
```

Comment:
re2c and gperf go to nativeBuildInputs too- according to other packages on Nixpkgs